code/modules/mob/living/silicon/ai/vox_sounds.dm

```diff
@@ -553,19 +861,24 @@ GLOBAL_LIST_INIT(vox_sounds, list("!bloop" = 'sound/vox/!bloop.ogg',
 		"threat" = 'sound/vox/threat.ogg',
 		"three" = 'sound/vox/three.ogg',
 		"through" = 'sound/vox/through.ogg',
+		"throwing" = 'sound/vox/throwing.ogg',
 		"time" = 'sound/vox/time.ogg',
-		"to" = 'sound/vox/to.ogg',
+		"to" = 'sound/vox/two.ogg',
```

Comment:
??

Reply:
This was a size saving measure, I can generate different sound files if you need them to have seperate .ogg files.